pages/linux/netcat.md

```diff
@@ -0,0 +1,36 @@
+# netcat
+
+> The `nc` (or `netcat`) utility is used for just about anything under the sun involving TCP, UDP, or UNIX-domain sockets. It can open TCP connections, send UDP packets, listen on arbitrary TCP and UDP ports, do port scanning.
+> More information: <http://man.openbsd.org/OpenBSD-current/man1/nc.1>.
+
+- Normal Syntax:
+
+`nc {{options}} {{host}} {{port}}`
+
+- Use IPv4 addressing only:
+
+`nc -4 {{host}} {{port}}`
+
+- Use IPv6 addressing only:
+
+`nc -6 {{host}} {{port}}`
+
+- UDP instead of TCP:
+
+`nc -u {{host}} {{port}}`
+
+- Use random port number:
+
+`nc -r {{host}}`
+
+- Listen for an incoming connection rather than initiate connection:
+
+`nc -l {{host}} {{port}}`
```

Comment:
```suggestion
- [l]isten for an incoming connection rather than initiate connection:

`nc -l "{{host}}" "{{port}}"`
```

The same issue as above.